core/src/main/scala-2.13+/cats/data/NonEmptyLazyList.scala

```diff
@@ -362,8 +362,23 @@ sealed abstract private[data] class NonEmptyLazyListInstances extends NonEmptyLa
     Semigroup[LazyList[A]].asInstanceOf[Semigroup[NonEmptyLazyList[A]]]
 
   implicit def catsDataShowForNonEmptyLazyList[A](implicit A: Show[A]): Show[NonEmptyLazyList[A]] =
-    Show.show[NonEmptyLazyList[A]](nec => s"NonEmpty${Show[LazyList[A]].show(nec.toLazyList)}")
+    Show.show[NonEmptyLazyList[A]](nell => s"NonEmpty${Show[LazyList[A]].show(nell.toLazyList)}")
 
+  implicit def catsDataParallelForNonEmptyLazyList: Parallel[NonEmptyLazyList, OneAnd[ZipLazyList, *]] =
+    new Parallel[NonEmptyLazyList, OneAnd[ZipLazyList, *]] {
+
+      def applicative: Applicative[OneAnd[ZipLazyList, *]] =
+        OneAnd.catsDataApplicativeForOneAnd(ZipLazyList.catsDataAlternativeForZipLazyList)
```

Comment:
I think this is the wrong `Applicative`, no? It's `pure` should be `LazyList.continually`.

Reply:
@LukaJCB It's the same as the one used in the `Parallel` instance for `NonEmptyStream`, and works the same way:

```scala
scala> import cats.data._
import cats.data._

scala> val F = OneAnd.catsDataApplicativeForOneAnd(ZipLazyList.catsDataAlternativeForZipLazyList)
F: cats.Applicative[[β$22$]cats.data.OneAnd[[A]cats.data.ZipLazyList[A],β$22$]] = cats.data.OneAndLowPriority2$$anon$8@3e84b276

scala> F.pure(1).unwrap.value.take(10).toList
res0: List[Int] = List(1, 1, 1, 1, 1, 1, 1, 1, 1, 1)
```

Reply:
(Which is just to say I'm pretty sure it's the right one, and its `pure` does return an infinite `OneAnd`.)

Reply:
Ah great, sorry about that. I guess I'm a bit confused as `catsDataApplicativeForOneAnd ` looks like this: https://github.com/typelevel/cats/blob/48c8b344aaed1f796aebbfdfa610432bee641d84/core/src/main/scala/cats/data/OneAnd.scala#L234

Reply:
@LukaJCB Right, the `continually`-ing happens in `unwrap`.